pkg/controller/cloneset/cloneset_controller.go

```diff
@@ -237,14 +237,15 @@ func (r *ReconcileCloneSet) doReconcile(request reconcile.Request) (res reconcil
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+	// filteredPVCS's ownerRef is CloneSet
 	filteredPVCs, err := r.getOwnedPVCs(instance)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
 
-	// If mcloneset doesn't want to reuse pvc, clean up
-	// the existing pvc first. Then it looks like the pod
-	// is deleted by controller, new pod can be created.
+	// If cloneSet doesn't want to reuse pvc, clean up
+	// the existing pvc first, which owned by inactive pods.
```

Comment:
which owned by inactive pods -> which are owned by inactive or deleted pods. 
> Then it looks like the pod is deleted by controller, new pod can be created. 
just delete this comment which is quite confusing